src/main/java/com/example/controller/GraphController.java

```diff
@@ -55,14 +69,31 @@ public void removeObserver(GraphObserver<Integer, Integer> observer) {
         observers.remove(observer);
     }
 
-    public void setModelGraph(Graph<Integer, Integer> graph){
+    private void updateCss() {
```

Comment:
I think this method can be removed since I've already fixed coloring in another story.

Reply:
done